tools/extra/fpgabist/bist_common.py

```diff
@@ -127,18 +129,25 @@ def get_mode_from_path(gbs_path):
 
 
 def load_gbs(gbs_file, bdf):
-    print "Attempting Partial Reconfiguration:"
+    print("Attempting Partial Reconfiguration:")
     cmd = ['fpgaconf', '-B', hex(bdf['bus']), '-D',
            hex(bdf['device']), '-F', hex(bdf['function']),
-           gbs_file]
+           '-v', gbs_file]
```

Comment:
-v will give the version and exit.  I think you want -V.

Reply:
changes to  -V , Fixed